### DISCUSSION TITLE
Unexpected Connection Error dialog behavior

### DISCUSSION THREAD

**Discussion by ericwb** [reflex-dev/reflex#3297] [General]
I realize Reflex probably designed to maintain a constant connection so the backend can serve the site. However, when viewing pages such as the Reflex documentation, I didn't expect to see a Connection Error dialog whenever my wifi drops. What's worse, is that I can't scroll or navigate the page when this dialog appears. 

I haven't seen any website behave like this except sites that are expected to have an active connection open. For example, if I use a web site that opens remove console of a remote system (think VNC in the browser), I would expect a dropped connection dialog when wifi is down.

In the case of Reflex, the [docs site](https://reflex.dev/docs/library/datadisplay/datatable/) will throw a connection error. I think of a docs site as more of a static single page application. And if I ever wanted to read Reflex docs on an airplane, this wouldn't work.

<img width="1045" alt="Screenshot 2024-05-13 at 7 08 56 PM" src="https://github.com/reflex-dev/reflex/assets/5316833/eda30b16-4c19-4d41-90fa-917fabe4bdb1">

**Comment by masenf** [comment]
Yeah this has been a persistent issue. We will be replacing the connection banner with a toast in a future release, that should be less invasive.
